packages/big-design/src/components/Collapse/spec.tsx

```diff
@@ -21,150 +23,159 @@ const CollapseWithVisiblePanelMock = (
 );
 
 test('renders with title button', () => {
-  const { getByRole } = render(CollapseWithStaticTitleMock);
+  render(CollapseWithStaticTitleMock);
 
-  expect(getByRole('button')).toBeInTheDocument();
+  expect(screen.getByRole('button')).toBeInTheDocument();
 });
 
 test('renders with provided title', () => {
-  const { queryAllByText } = render(CollapseWithStaticTitleMock);
+  render(CollapseWithStaticTitleMock);
 
-  expect(queryAllByText('title').length).toBe(2);
+  expect(screen.getAllByText('title')).toHaveLength(2);
 });
 
 test('renders with panel', () => {
-  const { getByRole } = render(CollapseWithVisiblePanelMock);
+  render(CollapseWithVisiblePanelMock);
 
-  expect(getByRole('region')).toBeInTheDocument();
+  expect(screen.getByRole('region')).toBeInTheDocument();
 });
 
 test('title button has id', () => {
-  const { getByRole } = render(CollapseWithStaticTitleMock);
+  render(CollapseWithStaticTitleMock);
 
-  expect(getByRole('button').id).toBeDefined();
+  expect(screen.getByRole('button').id).toBeDefined();
 });
 
 test('title button has aria-controls', () => {
-  const { getByRole } = render(CollapseWithStaticTitleMock);
-  const button = getByRole('button');
-  const panel = getByRole('region', { hidden: true });
+  render(CollapseWithStaticTitleMock);
 
-  fireEvent.focus(button);
+  const button = screen.getByRole('button');
+  const panelId = screen.getByRole('region', { hidden: true }).id;
 
-  expect(button.getAttribute('aria-controls')).toBe(panel.id);
+  expect(button.getAttribute('aria-controls')).toBe(panelId);
 });
 
 test('title button has icon', () => {
-  const { getByRole } = render(CollapseWithStaticTitleMock);
-  const trigger = getByRole('button');
-  const icon = trigger.querySelector('svg');
+  render(CollapseWithStaticTitleMock);
+
+  const icon = screen.getByRole('button').querySelector('svg');
```

Comment:
🍹 Using `querySelector` to get the SVG is a little concerning. It might mean that we need to add a role to the image for accessibility, potentially along with any additional aria tags 🤔 

No action needed for this, just pointing out that if we are using `querySelector` something might be wrong.